addons/confirm-actions/userscript.js

```diff
@@ -10,7 +10,11 @@ export default async function ({ addon, console, msg }) {
         cancelMessage = msg("share");
       } else if (addon.settings.get("followinguser") && e.target.closest("#profile-data .follow-button")) {
         cancelMessage = msg("follow");
-      } else if (addon.settings.get("joiningstudio") && e.target.closest("a.accept")) {
+      } else if (
+        addon.settings.get("joiningstudio") &&
+        (e.target.closest("a.accept") ||
+          e.target.textContent === addon.tab.scratchMessage("studio.curatorAcceptInvite"))
```

Comment:
This is truthy when clicking anything without a textContent (for example an image) in any non-studio page without that locale (for example, explore page before new studios were shipped - not sure if it will have the locale after it)